packages/cli/src/commands/transfer/celo.ts

```diff
@@ -22,42 +23,80 @@ export default class TransferCelo extends BaseCommand {
   ]
 
   async run() {
-    const kit = await this.getKit()
+    const client = await this.getPublicClient()
+    const wallet = await this.getWalletClient()
     const res = await this.parse(TransferCelo)
 
+    if (!wallet) {
+      throw new Error('TODO: only AKV doesnt return a wallet')
```

Comment:
I think we can just throw here with "useAVK flag is no longer supported on transfer commands"

Reply:
for context multiply validator operators said they dont use it. none said they do

Reply:
we should include the break in the change notes though